acs-aem-commons/pages/maven.md

```diff
@@ -14,73 +14,126 @@ Please refer to [Compatibility](/acs-aem-commons/pages/compatibility.html) to ch
 
 ## ACS AEM Commons 6.0.0+
 
-### AEM 6.5
+*In ACS AEM Commons 6.0.0, the main dependency artifact ID was renamed from `acs-aem-commons-content` to `acs-aem-commons-all`.*
 
-In ACS AEM Commons 6.0.0, the main dependency artifact ID was renamed from `acs-aem-commons-content` to `acs-aem-commons-all`. 
+The setup for embedding/depending on ACS AEM Commons in your code base differs slightly between AEMaaCS and AEM 6.5.
 
-#### Your all/pom.xml
+### AEM as a Cloud Service
 
-In the `filevault-package-maven-plugin` plugin configuration of your _all project's pom.xml_ file, add this:
+#### Your Container Content Package (usually called `all/pom.xml `)
+
+In your _container content package module's pom.xml_ file you extend both the
+
+* `dependencies` section and the
+* `filevault-package-maven-plugin` configuration
+
+like this:
 
 {% highlight xml %}
 <plugins>
-    <plugin>
-        <groupId>org.apache.jackrabbit</groupId>
-        <artifactId>filevault-package-maven-plugin</artifactId>
+  <plugin>
+    <groupId>org.apache.jackrabbit</groupId>
+    <artifactId>filevault-package-maven-plugin</artifactId>
+      ...
+      <configuration>
+        <embeddeds>
+          <embedded>
+            <!-- the artifact with the given id from the project dependencies is embedded -->
+            <artifactId>acs-aem-commons-all</artifactId>
```

Comment:
@kwin Don't we need <groupId>?

Reply:
No, this acts as filter on the project dependencies.

Reply:
@kwin I see the bundles also getting installed with the dependency defined in all's pom.xml. Do we need the core dependencies then?

<img width="1329" alt="image" src="https://github.com/Adobe-Consulting-Services/adobe-consulting-services.github.io/assets/86303251/0bf0b7df-278e-4bf5-a1c1-8db3defcfec0">

Reply:
all is a container comprising everything relevant.

Reply:
@kwin Then we need not have the optional dependency in core/pom.xml as all/pom.xml will always deploy the bundles.

Reply:
Core != All. The former is an OSGi bundle which needs the ACS AEM Commons bundle as dependency to appear in the compile time class path in order to use ACS Commons API.

Reply:
Thanks for clarifying @kwin. I was only thinking runtime since I noticed the bundles installed. Thanks again!